modules/ROOT/pages/js-movie-app.adoc

```diff
@@ -398,10 +448,62 @@ curl -X POST --header 'Content-Type: application/json' --header 'Accept: applica
 {}
 ----
 
-.JavaScript Implementation: Rate a Movie
+The user sends who they are, what movie they want to rate, and what their rating is through the endpoint. 
+
+.api/app.js
+[source,javascript,role=nocopy]
+----
+api.post("/movies/:id/rate", routes.movies.rateMovie);
+----
+
+The information is routed through `api/routes/movies.js`:
+
+.api/routes/movies.js
 [source,javascript,role=nocopy]
 ----
-// TK JS Rate A Movie
+exports.rateMovie = function (req, res, next) {
+  loginRequired(req, res, () => {
+    var rating = Number(_.get(req.body, 'rating'));
+    if (isNaN(rating) || rating < 0 || rating >= 6) {
```

Comment:
nit: `Number.isNaN` is known to be a bit less surprising with some edge cases than the global `isNaN`